package.json

```diff
@@ -83,7 +84,10 @@
     "start-static": "npm run compile:prod && tt-serve-static --site ./build",
     "test": "npm run jest && npm run wdio",
     "update-package-readme": "node scripts/root-readme-updater/index.js",
-    "wdio": "wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js"
+    "wdio-default": "wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
+    "wdio-fusion": "THEME=orion-fusion-theme wdio node_modules/terra-toolkit/config/wdio/wdio.conf.js",
+    "wdio": "npm run wdio-default && npm run wdio-fusion",
+    "tt-wdio": "tt-wdio --config node_modules/terra-toolkit/config/wdio/wdio.conf.js"
```

Comment:
Shouldn't this pass the themes options for local testing? ```tt-wdio --config node_modules/terra-toolkit/config/wdio/wdio.conf.js --themes=['default,'fusion']```

Reply:
This command was just to dry up the travis yml